### PR TITLE
new sidebar with responsive header and sidebar

### PR DESCRIPTION
**Pull request by Souad-Sud**
Please review and test so we can merge this to main. There is some changes in header and footer and sdebar so just accept changes

### CODE REVIEW

**Comment by Souad-Sud** [comment]
And please do not add anything eles to the header and the footer its fixed

**Review by ranjithavelusamy — APPROVED** [review]
fine

File: scripts/sidebar.js
Comment (bushra-rauf) [inline scripts/sidebar.js:1]: What is this for. please delete your code. you have already choosen your page to do your js. On Miro board my name is on it.

**Review by bushra-rauf — CHANGES_REQUESTED** [review]

File: scripts/sidebar.js
Comment (bushra-rauf) [reply scripts/sidebar.js:1]: The animal name coming on page. I am working on it

Comment (bushra-rauf) [reply scripts/sidebar.js:1]: ok i will change it later. i am approving

**Comment by Moshira89** [comment]
The animal list needs fixing.

**Review by bushra-rauf — APPROVED** [review]

**Pull request merged**